pallets/pallet-asset-switch/README.md

```diff
@@ -0,0 +1,129 @@
+
+```toml
+pallet-asset-switch = {git = "https://github.com/KILTprotocol/kilt-node.git", branch = "release-1.13.0"}
+```
+
+The asset switch pallet is available in the KILT node release 1.13.0 and later.
+
+## Configure the pallet
+
+The pallet can be added one or more times to the runtime.
+
+For multiple deployments of the same pallet (e.g., to bridge the local currency to different remote assets), pass runtime configuration to the pallet's `Config` trait.
+
+```rust,ignore
+pub type SwitchPool1 = pallet_asset_switch::Instance1;
+impl pallet_asset_switch::Config<SwitchPool1> for Runtime {
+	// Config
+}
+
+pub type SwitchPool2 = pallet_asset_switch::Instance2;
+impl pallet_asset_switch::Config<SwitchPool2> for Runtime {
+	// Config
+}
+```
+
+If a single instance is required, then simply use the default instance:
+
+```rust
+impl pallet_asset_switch::Config for Runtime {
+    // Config
+}
+```
+
+## The `Config` trait
+
+As the pallet is generic over the runtime specifics, the `Config` trait requires the following configuration parameters passed to it:
+
+- `type AccountIdConverter: TryConvert<Self::AccountId, Junction>`: Because the `AccountId` type can be anything in the runtime, the converter is responsible for converting such a `AccountId` into a `Junction`, which is then used for some XCM processing.
+- `type AssetTransactor: TransactAsset`: This component is used when charging the extrinsic submitter with the XCM fees that the chain will pay at destination. For instance, if the transfer on the remote chain will cost 0.1 DOTs, the `AssetTransactor` might deduct 0.1 DOTs from the user's previously topped up balance on the source chain (more details below).
+- `type FeeOrigin: EnsureOrigin<Self::RuntimeOrigin>`: The origin that can update the XCM fee to be paid for the transfer on destination.
+- `type LocalCurrency: MutateFungible<Self::AccountId>`: The chain's local currency.
+- `type PauseOrigin: EnsureOrigin<Self::RuntimeOrigin>`: The origin that can pause a switch pair, e.g., if a vulnerability is found.
+- `type RuntimeEvent: From<Event<Self, I>> + IsType<<Self as frame_system::Config>::RuntimeEvent>`: The aggregate `Event` type.
+- `type SubmitterOrigin: EnsureOrigin<Self::RuntimeOrigin, Success = Self::AccountId>`: The origin that can call the `switch` extrinsic and perform the switch.
+- `type SwitchHooks: SwitchHooks<Self, I>`: Any additional runtime-specific logic that can be injected both before and after local tokens are exchanged for the remote assets, and before and after the remote assets are being converted into local tokens.
+- `type SwitchOrigin: EnsureOrigin<Self::RuntimeOrigin>`: The origin that can set, resume, and delete a switch pair.
+- `type XcmRouter: SendXcm`: The component responsible for routing XCM messages to the switch pair remote location to perform the remote asset transfer from the chain's sovereign account to the specified beneficiary.
```

Comment:
Is this the component, or does the comment relate to an XCM component?

Reply:
This is the component.